<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow4</class>
 <widget class="QMainWindow" name="MainWindow4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>425</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
 background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 #ffffff,stop:1 #ffffff);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
 background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 #91c46c,stop:1#91c46c);
 
color: white; </string>
    </property>
    <property name="text">
     <string>申请维修</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 #f2f2f2,stop:1 #f2f2f2);
 </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入您的姓名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入您的订单编号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 #f2f2f2,stop:1 #f2f2f2);
 </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>请选择您的故障类型：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 #f2f2f2,stop:1 #f2f2f2);
 </string>
    </property>
    <item>
     <property name="text">
      <string>家电外壳受损</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>因故障不能正常使用</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>上门服务</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>预计维修日期：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 #f2f2f2,stop:1 #f2f2f2);
 </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>271</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>预估费用：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 #f2f2f2,stop:1 #f2f2f2);
 </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/pic/10.png);


 background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 #91c46c,stop:1#91c46c);
 
}
QPushButton:pressed {
background-color: #287d7d;
}

 </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {
background-color: #91c46c;

}

QPushButton {
background-color: #ffffff;
color: #ffffff;
color: black;
                    /* 文字颜色 */
background-color: #f2f2f2;       /* 背景色 */
border-style: outset;            /* 边框风格 */
border-width: 2px;               /* 边框宽度 */
border-radius: 5px;              /* 边框圆角 */
border-color: #c6e070;           /* 边框颜色 */
font: bold 14px;                 /* 字体 */
min-width: 2em;                  /* 控件最小宽度 */
min-height: 2px;                 /* 控件最小高度 */
font-family: 宋体;            /* 字库 */
padding: 5px;                    /* 内边距 */
padding-left:6px;                /* 文字横向偏移 */
padding-top:6px;                 /* 文字纵向偏移 */
border-radius:15px
}
QPushButton:pressed {
background-color: #287d7d;
}
</string>
    </property>
    <property name="text">
     <string>提交申请</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>370</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
 background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 #f2f2f2,stop:1 #f2f2f2);
 </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>370</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
 background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 #1c344c,stop:1#1c344c);
 </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
 background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 #91c46c,stop:1#91c46c);
 </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>370</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
 background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 #287d7d,stop:1#287d7d);
 </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
 background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 #c6e070,stop:1#c6e070);
 </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>485</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
